<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenConfig</class>
 <widget class="QWidget" name="FenConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <property name="labelAlignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Code opérateur :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="opCode">
      <property name="text">
       <string>AB123</string>
      </property>
      <property name="maxLength">
       <number>5</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nb pièces/carton :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="nbPPCarton">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nb cartons/palette :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="nbCPalette">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Nombre de palettes de T1 :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="nNbT1">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Nombre de palettes de T2 :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="nNbT2">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttonLayout">
    <item>
     <widget class="QPushButton" name="bOk">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bCancel">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>formLayoutWidget</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>bOk</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bCancel</sender>
   <signal>clicked()</signal>
   <receiver>FenConfig</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
